### ISSUE TITLE
Provide package without groovy console embedded

### ISSUE THREAD

**Issue by royteeuwen** [valtech/aem-easy-content-upgrade#197]
I would like to start forking the ICF Next Groovyconsole as to be able to make improvements to it. AECU embeds the Groovyconsole it it's cloud package, making it hard to do this. 

Are you open to creating a package with and a package without the groovy console embedded? I don't mind contributing it myself but rather not do the effort if you aren't open to it :)

**Comment by gruberrolandvaltech** [comment]
To be honest, it looks like GroovyConsole is no longer maintained. We already have to repackage parts of it to get it running on AEM Cloud.

So if you release a cloud-compatible version with more features we are open to replace the ICF Next one.

**Comment by bdhoine** [comment]
@gruberrolandvaltech I linked the PR to update to the fork @royteeuwen is mentioning. Version 18.0.0 contains some new features and we have some other plans for the project (eg Groovy version update, tweak the UI, code base optimisations, ..). For now I added the runtime dependency to limit to <= 19. If there is a new major version available we will create a new PR and this time it will (should) only be a version bump 😀 and some end user testing of course.
